aten/src/ATen/native/IndexingUtils.h

```diff
@@ -15,6 +15,10 @@ static std::vector<Tensor> expandTensors(const Tensor & self, TensorList indices
   std::vector<Tensor> result;
   for (const auto & index : indices) {
     if (index.scalar_type() == kByte || index.scalar_type() == kBool) {
+      if (index.scalar_type() == kByte) {
+        AT_WARN("indexing with dtype torch.uint8 is now deprecated," \
```

Comment:
Wouldn’t it be appropriate to use TORCH_WARN instead of the deprecated AT_WARN?

Reply:
Its a cherry-pick from the master branch. If this has to be fixed, it should be done there first.